src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -254,6 +256,25 @@ private Step CreateStep(Step jsonStepInfo, HostControlInfo hostControlInfo, int
         private void Popup_StepClosed(string name, Step.StepTypes stepType)
         {
             GuideFlowEvents.OnGuidedTourFinish(currentGuide.Name);
+
+            //The exit tour popup will be shown only when a popup (doesn't apply for survey) is closed or when the tour is closed. 
+            if(stepType != Step.StepTypes.SURVEY)
+                CreateRealTimeInfoWindow();
+        }
+
+        private void CreateRealTimeInfoWindow()
+        {
+            //Search a UIElement with the Name "statusBarPanel" inside the Dynamo VisualTree
+            UIElement hostUIElement = Guide.FindChild(mainRootElement, "statusBarPanel");
+
+            //Creates the RealTimeInfoWindow popup and set up all the needed values to show the popup over the Dynamo workspace
+            var exitTourPopup = new RealTimeInfoWindow();
+            exitTourPopup.VerticalOffset = 30;
+            if (hostUIElement != null)
+                exitTourPopup.PlacementTarget = hostUIElement;
+            exitTourPopup.Placement = PlacementMode.Center;
```

Comment:
Hmm why would the placement be center?

Reply:
@QilongTang as I was saying in a comment below: "the code is looking for the "statusBarPanel" UIElement inside the Dynamo VisualTree, then this control will be the host (placement target) of the popup", then by saying that the Placement = Center means that the popup will be shown at the center of the host element. 
I think in the future the guided tour developers will need to have some knowledge of WPF Popups location and knowledge about the elements in the Dynamo VisualTree, because that knowledge will be needed when locating a popup in a specific location and using the offsets.